integration/integration_test.go

```diff
@@ -0,0 +1,64 @@
+package integration
+
+import (
+	"testing"
+
+	"sigs.k8s.io/krew/integration/krew"
+)
+
+const (
+	// validPlugin is a valid plugin with a small download size
+	validPlugin = "konfig"
+)
+
+func TestInstall(t *testing.T) {
```

Comment:
Hmm before these, maybe we should try more fundamental things like exit codes, -h/—help working, unknown subcmds failing, Wdyt? I can go ahead and merge this, and iterate later as well.